run.py

```diff
@@ -40,4 +46,5 @@ def config_validator():
         print("Starting server in ~*TEST MODE*~")
         app.run()
     else:
+        print("[Use http://localhost:5000/]")
```

Comment:
rha… my bad. It was a remaining of a previous test.
You may erase the start message.